client/public/index.html

```diff
@@ -46,14 +46,11 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
-  <!-- Compiled and minified JavaScript -->
```

Comment:
we still need this for any of the materialize js to work (even if jquery CDN is already there).

Reply:
no we don't. the autoinit does it. at least from everything i've seen. in fact this caused an error with materialize because we were calling it twice and our buttons wouldn't go back to their original state.

https://github.com/Dogfalo/materialize/issues/2667